### PR TITLE
[ICDS Dashboard] username regex issue dashbaord usage

### PR DESCRIPTION
**Pull request by challabeehyv**
##### SUMMARY
Fixed the username fiter regex issue

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/sqldata/exports/dashboard_usage.py [hunk 1]
```diff
@@ -186,7 +186,10 @@ def get_excel_data(self):
 
             dashboard_uname_rx = re.compile(r'^\d*\.[a-zA-Z]*@.*')
 
-            usernames = [user['username'] for user in users if dashboard_uname_rx.match(user['username'])]
+            usernames = []
```
Comment (Rohit25negi) [inline custom/icds_reports/sqldata/exports/dashboard_usage.py]: hey, the new code is also doing the same thing right.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]

File: custom/icds_reports/sqldata/exports/dashboard_usage.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: The earlier code is bypassing the condition

Comment (Rohit25negi) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: @challabeehyv

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@challabeehyv  can you

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**